Doc/Tutorial/chapter_contributing.tex

```diff
@@ -4,13 +4,11 @@ \section{Bug Reports + Feature Requests}
 
 Getting feedback on the Biopython modules is very important to us. Open-source projects like this benefit greatly from feedback, bug-reports (and patches!) from a wide variety of contributors.
 
-The main forums for discussing feature requests and potential bugs are the
-\href{http://biopython.org/wiki/Mailing_lists}{Biopython mailing lists}:
+The main forum for discussing feature requests and potential bugs is the
+\href{http://biopython.org/wiki/Mailing_lists}{Biopython mailing list}:
```

Comment:
I wonder if we should say something like this instead:

> The main forums for discussing feature requests and potential bugs are the
> \href{http://biopython.org/wiki/Mailing_lists}{Biopython mailing list} and
> issues or pull requests on GitHub.

What do you think?

Reply:
Good idea.

Reply:
I think we should mention an explicit mailing list e-mail address biopython@biopython.org after Biopython mailing list reference, either in parentheses or as a footnote to satisfy users who prefer the document in printed form.

Reply:
Maybe. The mailing list is subscriber only, and now ignored emails from non-subscribers silently (sadly necessary due to the spam load otherwise), so I'd make the URL which has this warning more prominent than the email address itself?

Reply:
Understand. We should mention only reference to Mailing list wiki without mailing list e-mail address biopython@biopython.org.